src/taskgraph/generator.py

```diff
@@ -66,6 +66,7 @@ def load_tasks(self, parameters, kind_dependencies_tasks, write_artifacts):
             config,
             parameters,
             list(kind_dependencies_tasks.values()),
+            write_artifacts,
```

Comment:
I guess it's not worth trying to avoid breaking backwards compat here?

Reply:
If there's a way to avoid the compat break without jumping through major hoops I'd love it...I couldn't find a way myself.

Reply:
Clever! Yes, let's do that!